utils/joiner/joinpartition.h

```diff
@@ -137,7 +149,7 @@ class JoinPartition
   uint64_t largeSizeOnDisk;
   utils::Hasher_r hasher;
   bool rootNode;
-
+  bool canSplit;
```

Comment:
According with the style guide we add a suffix '_' to an attribute,e.g canSplit_.